README.md

```diff
@@ -38,7 +40,7 @@ Sphinx is currently invite-only. [Request access on our website.](https://sphinx
 - [CLI Propose Command](https://github.com/sphinx-labs/sphinx/blob/main/docs/cli-propose.md)
 - [Propose in CI](https://github.com/sphinx-labs/sphinx/blob/main/docs/ci-proposals.md)
 - [Configuration Options](https://github.com/sphinx-labs/sphinx/blob/main/docs/configuration-options.md)
-- [Deploy from the CLI](https://github.com/sphinx-labs/sphinx/blob/main/docs/cli-deployments.md)
+- [Deploy from the CLI](https://github.com/sphinx-labs/sphinx/blob/main/docs/cli-deploy.md)
```

Comment:
Changed the file name from 'cli-deployments' to 'cli-deploy' to align with the proposal CLI guide (`cli-propose.md`)